### PR TITLE
Feature single product page

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I added single Product page. #26 
-user can navigate over images for product detailings.
-user can add product to wishlist and cart.

Screenshot (1411)

### CODE REVIEW

File: src/Pages/Home/Home.css [hunk 1]
```diff
@@ -1,47 +1,261 @@
...
+  position: absolute;
+  top: 50%;
+  left: 80%;
+  transform: translate(-80%, -50%);
   text-align: center;
-  color: var(--dark-black);
+  width: 30%;
+}
+.video-title {
+  color: #14281d;
```
Comment (ajit-sharmaGH) [inline src/Pages/Home/Home.css:21]: var color

File: src/index.css [hunk 2]
```diff
@@ -1,11 +1,25 @@
 @import url("/node_modules/mockman-js/dist/style.css");
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600&display=swap");
 :root {
   --black-color: black;
+  --dark-green: #4b5563;
+  --grey-color: #9ca3af;
+  --white-color: white;
```
Comment (ajit-sharmaGH) [inline src/index.css:7]: `#FFFFFF`

File: src/Components/FilterSidebar/FilterSidebar.css [hunk 3]
```diff
@@ -40,46 +44,87 @@
...
+@media screen and (max-width: 1250px) {
+  .drawer {
+    width: 25%;
+  }
+}
+@media screen and (max-width: 900px) {
+  /* .drawer-container {
+    display: flex;
+    position: fixed;
+    inset: 0;
```
Comment (ajit-sharmaGH) [inline src/Components/FilterSidebar/FilterSidebar.css:91]: if(no use further) 
can remove
else
ignore it!

File: src/backend/db/products.js [hunk 4]
```diff
@@ -13,6 +13,14 @@ export const products = [
...
     modelImg:
       "https://ik.imagekit.io/yol3sixl2xj/product-listing/model9_Oh9hTgpxE8.jpg?ik-sdk-version=javascript-1.4.3&updatedAt=1649654103792",
+    imgOne:
+      "https://res.cloudinary.com/bhakti1801/image/upload/v1649913101/1-1_n0zr8v.webp",
+    imgTwo:
+      "https://res.cloudinary.com/bhakti1801/image/upload/v1649913099/1-2_wkqt5o.webp",
+    imgThree:
+      "https://res.cloudinary.com/bhakti1801/image/upload/v1649913100/1-3_rxsif3.webp",
+    imgFour:
+      "https://res.cloudinary.com/bhakti1801/image/upload/v1649913099/1-4_hxb7gc.webp",
```
Comment (tushartiwari7) [inline src/backend/db/products.js]: Suggestion: You may use an Array of images here as you have multiple images.

**Review by ajit-sharmaGH — APPROVED** [review]
Everything used systematically
Code structures are good 
Better naming convention
Blocker - too big files

**Review by tushartiwari7 — APPROVED** [review]
Ui looks very beautiful and impressive.
Just a small suggestion, please look into it.

**Pull request merged**